iconservice/iconscore/icon_system_score_base.py

```diff
@@ -53,11 +53,19 @@ def __init__(self, db: 'IconScoreDatabase') -> None:
     def is_builtin_score(self, score_address: 'Address') -> bool:
         return util_is_builtin_score(str(score_address))
 
+    # TODO remove after Update 0.0.6
+    def get_icon_service_flag(self) -> int:
+        return IconScoreContextUtil.icon_service_flag
+
+    def get_icon_score_context_util(self) -> Type['IconScoreContextUtil']:
```

Comment:
I think that IconScoreContextUtil should be used in IconScoreBase or IconSystemScoreBase classes only.
IconScoreContextUtil should not be exposed to SCORE subclasses directly.

Reply:
I understand
first of all, removed get_icon_score_context_util function and made specific functions to replace logics which use get_icon_score_context_util